bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ChallengeResponse/TallyChallengeResponse.cs

```diff
@@ -0,0 +1,113 @@
+using ElectionGuard.Ballot;
+using ElectionGuard.Decryption.Challenge;
+using ElectionGuard.Decryption.Shares;
+using ElectionGuard.Decryption.Tally;
+using ElectionGuard.ElectionSetup;
+using ElectionGuard.ElectionSetup.Extensions;
+using ElectionGuard.Guardians;
+
+namespace ElectionGuard.Decryption.ChallengeResponse;
+
+/// <summary>
+/// The Challenge response for a specific tally
+/// </summary>
+public record TallyChallengeResponse
+    : DisposableRecordBase, IEquatable<TallyChallengeResponse>
+{
+    // TODO: tallyId
```

Comment:
I'm not sure if we need to propagate the tally id throughout all of these objects or not. maybe we do. i left this todo here as a note but we should make that decision. i think we probably need it so that we can query the database by tally but im not sure. what do you guys think?

Reply:
added